<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="tb3_0">
        <param name="tf_prefix" value="tb3_0" />
        <include file="$(find turtlebot3_bringup)/launch/turtlebot3_remote.launch">
            <arg name="multi_robot_name" value="tb3_0" />

        </include>
    </group>

    <group ns="tb3_1">
        <param name="tf_prefix" value="tb3_1" />
        <include file="$(find turtlebot3_bringup)/launch/turtlebot3_remote.launch">
            <arg name="multi_robot_name" value="tb3_1" />
        </include>
    </group>
</launch>